app/frontend/actions.ts

```diff
@@ -1110,6 +1110,15 @@ export default ({setState, getState}: {setState: SetStateFn; getState: GetStateF
     })
   }
 
+  const getErrorBannerContent = (state) => {
+    const errorBannerContent = ADALITE_CONFIG.ADALITE_ERROR_BANNER_CONTENT
+    const showErrorBanner = errorBannerContent !== '' && true
```

Comment:
I would rather use `!!errorBannerContent` here - maybe not an issue now, but `errorBannerContent !== ''` would evaluate to true even if the content is undefined which seems like a chance to shoot ourselves in the foot